core/system/src/privcall.c

```diff
@@ -15,7 +15,18 @@
  * limitations under the License.
  */
 #include "api/inc/privcall_exports.h"
+#include <uvisor.h>
```

Comment:
It's needed because vmpu.h needs it for things like `FLASH_ORIGIN`. The better thing to do would be to make vmpu.h include the headers it needs instead.